src/containers/signupFlow/index.tsx

```diff
@@ -1,29 +1,144 @@
 import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
+import { useDispatch } from 'react-redux';
 import { Route } from 'react-router-dom';
+import { signup } from '../../auth/ducks/thunks';
 import { ContentContainer } from '../../components';
-import SignupDirectory from '../../components/SignupDirectory';
 import GMSignupForm from '../../components/GMSignupForm';
-import SignupVerification from '../../components/SignupVerification';
-import SignupConfirmation from '../../components/SignupConfirmationForm';
 import PF1SignupForm from '../../components/PF1SignupForm';
 import PF2SignupForm from '../../components/PF2SignupForm';
+import SignupConfirmation from '../../components/SignupConfirmationForm';
+import SignupDirectory from '../../components/SignupDirectory';
+import SignupVerification from '../../components/SignupVerification';
+import { encodeProfileFieldFile } from '../../utils/fileEncoding';
 
 const SignupFlow: React.FC = () => {
   const [GMForm, setGMForm] = useState<any | null>(null);
   const [PFForm1, setPFForm1] = useState<any | null>(null);
   const [PFForm2, setPFForm2] = useState<any | null>(null);
+  const dispatch = useDispatch();
 
-  const submitGMForm = (photoRelease: boolean) => {
-    // temporary console logs, just to show data is being collected correctly.
-    console.log(GMForm); // tslint:disable-line
-    console.log(photoRelease); // tslint:disable-line
+  function convertToYearMonthDateString(d: Date): string {
+    return d.toISOString().split('T')[0];
+  }
```

Comment:
This should go in a utility file and be tested, it's hard to understand what it is doing.